src/Elastic.Apm.AspNetCore/Extensions/RequestExtentions.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.IO;
+using System.Text;
+using Elastic.Apm.Logging;
+using Elastic.Apm.Model;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+
+namespace Elastic.Apm.AspNetCore.Extensions
+{
+    public static class HttpRequestExtensions
```

Comment:
is there a specific linting tool use in this project?

Reply:
Most of us working on this project use JetBrains Rider, here is how the file looks in it (spot the yellow things)

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/1091853/61944673-8efab780-af9e-11e9-9d18-450ae894eb0f.png">

I suspect you use Visual Studio, is that right? VS also shows some of the issues:

<img width="1138" alt="image" src="https://user-images.githubusercontent.com/1091853/61945177-a4241600-af9f-11e9-9661-e82926d3a141.png">

As you can see it shows the unused `using`s, plus I turned on showing whitespace characters, that's how I saw that spaces and tabs are mixed (as you can see... there are seom `. . . . ` lines and some `-> ->`.  Regarding the missing new lines: those are unfortunately not shown in VS, but if you reformat, VS takes care of those - same for the tabs vs. spaces issue.

I think the only thing I saw and VS did not find and fix is the `string body` instead of `var body`.

So.. the way this works is that we have an [`.editorconfig`](https://github.com/elastic/apm-agent-dotnet/blob/master/.editorconfig) file, and the editor picks that up. Everything is defined there.

Reply:
I verified all files are cleaned-up before my last commit from a few minutes ago.